addons/stock/views/product_views.xml

```diff
@@ -268,9 +268,19 @@
                 <div name="o_kanban_price" position="after">
                     <div t-if="record.is_storable.raw_value"
                          name="o_kanban_qty_available">
-                        <span>On Hand: </span>
                         <field name="qty_available"/>
-                        <field name="uom_id" widget="many2one_uom" class="ms-1" groups="uom.group_uom"/>
+                        <span class="ms-2"> On Hand </span>
+                        <span>
+                            (<span invisible="catalog_diff_qty != 0">
+                                <field name="catalog_diff_qty"/>
+                            </span>
+                            <span class="text-success" invisible="catalog_diff_qty &lt;= 0">
+                                +<field name="catalog_diff_qty"/>
+                            </span>
+                            <span class="text-danger" invisible="catalog_diff_qty &gt;= 0">
+                                <field name="catalog_diff_qty"/>
+                            </span>)
+                        </span>
```

Comment:
I don't understand why you removed the forecasted quantity. This is NOT the On Hand quantity as the label would suggest. Also, I'm not a big fan of adding one more computation to product. Why do you need that?